src/asciidoc/docs/publish/customDomains.adoc

```diff
@@ -0,0 +1,84 @@
+= Using custom domains
+:page-slug: custom-domains
+:page-description: How to configure your Koji to use a custom domain.
+
+When you're ready to go live with your template, you might want to use a custom domain name in place of the name provided by Koji.
+A custom domain can have the advantage of being easier to remember, and it usually just looks better.
+
+Koji has a few subdomains that you can use at no cost and with no additional configuration.
+These include:
+
+* withkoji.com
+* koji.sh
+* koji.game
+
+You can also use a domain name that you purchase from somewhere like *Google Domains* or *Namecheap*.
+
+== Using a Koji subdomain
+
+. Open the Koji and click the Koji button.
+
+. Go to *Settings > Custom domains*.
+
+. If you have already configured a domain and want to change it, follow the steps for <<_releasing_a_domain>> and then come back to this step.
+If you have not already configured a domain for this Koji, click *New*.
+
+. Select *Subdomain*.
+
+. Click *Next*.
+
+. Select a *root domain*.
+
+. Enter the name that you want to come before the subdomain.
+
+. Click *Save*.
+If the domain name you chose is already taken, "That domain is not available" will be displayed.
+If the domain name is available, the complete domain name will be displayed.
+
+. Click the domain name.
+You should see "Your Koji has been successfully configured to start receiving traffic from your custom domain".
+
+. Click *Open* to go to the new URL.
+Your Koji should display.
+
+== Using a domain you own
+
+ . Use your domain provider's management tools to configure the appropriate DNS records.
++
+[NOTE]
+If you want to host your Koji on a subdomain of your custom domain name (for example, *www.mycustomdomain.com or game.mycustomdomain.com*), you need to use your domain provider's tools to add a CNAME record for that subdomain (*www* or *game*) that points to *cnames.withkoji.com*.
+
+. Open the Koji and click the Koji button.
+
+. Go to *Settings > Custom domains*.
+
+. If you have already configured a domain and want to change it, follow the steps for <<_releasing_a_domain>> and then come back to this step.
+If you have not already configured a domain for this Koji, click *New*.
+
+. Select *A domain you own*.
+
+. Click *Next*.
+
+. Enter your complete domain name.
+
+. Click *save*.
+The domain name should be displayed.
+
+. Click the domain name.
+You should see "Your Koji has been successfully configured to start receiving traffic from your custom domain".
+
+. Click *Open* to go to the new URL.
+Your Koji should be displayed.
+
+== Releasing a domain
+
+. Open the Koji and click the Koji button.
+
+. Go to *Settings > Custom domains*.
+
+. Your complete domain name should appear.
```

Comment:
This is not written as a step. Steps describe an action, not a result. Try something like, "Click the domain name you want to release.